config/manager/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
```

Comment:
@arturdzm RCO doesn't have this. Wondering if that's because OLO was generated with a different version of operator-sdk. 

Do we need this? This results in `bundle/manifests/olo-manager-config_v1_configmap.yaml` being generated.